### PR TITLE
Add Angular button docs to Storybook

### PR DESCRIPTION
**Pull request by marclove**

### CODE REVIEW

**Comment by camwest** [comment]
@marclove question. Is it possible for a user to pass the attributes as strings or do they HAVE to be scope bindings?

File: packages/storybook/stories/button.js [hunk 1]
```diff
@@ -69,6 +69,45 @@ storiesOf('Button', module)
...
+            });
+
+          // ------------------------------
+          // index.html
+
+          <!doctype html>
+          <html>
+          <body ng-app="app">
+            <div ng-controller="DemoController as ctrl">
+              <orion-button background="ctrl.button.label" color="ctrl.button.color" ng-click="ctrl.action('clicked')">
```
Comment (camwest) [inline packages/storybook/stories/button.js]: Background being set to label seems wrong.

File: packages/storybook/stories/button.js [hunk 2]
```diff
@@ -69,6 +69,45 @@ storiesOf('Button', module)
           </Button>
         `,
       },
+      {
+        label: 'Angular 1.5.x',
+        source: `
+          // ------------------------------
+          // controller.js
+
+          import 'angular';
```
Comment (camwest) [inline packages/storybook/stories/button.js]: Do you mind including the import for the orion lib here too?

File: packages/storybook/stories/button.js (hunk 1 above)
Comment (marclove) [inline packages/storybook/stories/button.js]: Eeek! Sorry.

**Comment by marclove** [comment]
@camwest as long as attributes are evaluated by angular, they will work. so it doesn't have to be a scope binding, but it needs to be a value evaluated by angular in some way.

if it's not evaluated by angular, then it's just an html attribute on the native element. since we didn't provide attribute support on the native element (only property support), that would not work.

**Comment by camwest** [comment]
@marclove OK awesome. So we could add attribute support and then a straight up string attribute would work.

Thanks!

**Comment by camwest** [comment]
@marclove can you update the source examples to be the simplest examples possible. I'd like the examples to convey the smallest amount of code required to achieve the view state.

Other than that this change looks good.

**Pull request merged**